server/utils/auth/google_client.py

```diff
@@ -85,4 +87,37 @@ def get_user_calendar_info(self, timeMin, timeMax, timezone):
             return busy_time
 
         except Exception as e:
-            print(e)
\ No newline at end of file
+            print(e)
+
+    def create_event_type(self, duration, title, start_time, timezone, host_email, booker_email):
+        end_time = (dateutil.parser.isoparse(start_time) + timedelta(minutes=duration)).isoformat()
+        event = {
+            'summary': 'Appointment with CalendApp',
+            'description': title,
+            'start': {
+                'dateTime': start_time,
+                'timeZone': timezone,
+            },
+            'end': {
+                'dateTime': end_time,
+                'timeZone': timezone,
+            },
+            'recurrence': [
+                'RRULE:FREQ=DAILY;COUNT=1'
+            ],
+            'attendees': [
+                {'email': host_email},
+                {'email': booker_email},
+            ],
+            'reminders': {
+                'useDefault': False,
+                'overrides': [
+                    {'method': 'email', 'minutes': 24 * 60},
+                    {'method': 'popup', 'minutes': 10},
+                ],
+            },
+        }
+
+        event_service = build('calendar', 'v3', credentials=self.credentials)
+        event = event_service.events().insert(calendarId='primary', body=event).execute()
```

Comment:
@fangyu3 and please review this part, if no problem I'll merge then

Reply:
yep i see no issues! If you tested it and it works then it's good to go!